lib/xboxrt/wchar.c

```diff
@@ -0,0 +1,43 @@
+#include "wchar.h"
+
+#include <stdint.h>
+
+wchar_t *wcscpy(wchar_t *dst, const wchar_t *src) {
+    return wcsncpy(dst, src, SIZE_MAX);
+}
+
+wchar_t *wcsncpy(wchar_t *dst, const wchar_t *src, size_t n) {
+    size_t i = 0;
+    do dst[i] = *src; while (*src++ != L'\0' && i++ < n);
+    return dst;
```

Comment:
`man wcsncpy`

> Exactly n wide characters are written at dest. If the length wcslen(src) is smaller than n, the remaining wide characters in the array pointed to by dest are filled with null wide characters.

Another error that propagated: we currently don't fill exactly `n` bytes (and neither could we, because our users don't respect that - see `wcscpy` which calls with `SIZE_MAX`)

*Same as other instance: don't fix, we should fix this all at once.*